pkg/runtime/security/auth.go

```diff
@@ -153,8 +155,28 @@ func (a *authenticator) CreateSignedWorkloadCert(id, namespace, trustDomain stri
 	return signedCert, nil
 }
 
-// currently we support Kubernetes identities.
-func getToken() string {
+// Returns the token for authenticating with Sentry.
+func getToken() (token string, validator sentryv1pb.SignCertificateRequest_TokenValidator) {
+	// Check if we have a token in the DAPR_SENTRY_TOKEN env var (for the JWKS validator)
+	if v := os.Getenv(consts.SentryTokenEnvVar); v != "" {
```

Comment:
We must use this validator if the environment variable set. It's beside the point of the value has any string.

```suggestion
	if v, ok := os.LookupEnv(consts.SentryTokenEnvVar); ok {
```